include/GloryDuckTables2019Lkl.h

```diff
@@ -0,0 +1,74 @@
+//////////////////////////////////////////////////////////////////////
+// Binned flux likelihood
+//////////////////////////////////////////////////////////////////////
+
+#ifndef GLORYDUCKLKL
+#define GLORYDUCKLKL
+
+#include "TObjArray.h"
+#include "TCanvas.h"
+
+#include "Lkl.h"
+
+class GloryDuckTables2019Lkl : public Lkl
+{
+ public:
+  // constructors
+  GloryDuckTables2019Lkl(TString inputString="");
+  
+  // destructor
+  virtual ~GloryDuckTables2019Lkl();
+ 
+  // make the TObjArray own its contents
+  void SetOwner(Bool_t set=kTRUE)            {fSampleArray->SetOwner(set);}
+
+  // add samples
+  inline void AddSample(Lkl* sample)        {fSampleArray->Add(sample); SetChecked(kFALSE);}
+
+  // getters
+  inline const TObjArray* GetSampleArray()     const {return fSampleArray;}
+  inline       Int_t      GetNSamples()        const {return fSampleArray->GetEntries();}
+  inline       Lkl*       GetReferenceSample() const {return (Lkl*)fSampleArray->At(0);}
+  inline       Lkl*       GetSample(Int_t i)   const {return (Lkl*)fSampleArray->At(i);}
+  inline       Double_t   GetLogJ()            const {return fLogJ;}
+  inline       UInt_t     GetNMasses()         const {return fNMasses;}
+  inline       UInt_t     GetActiveMass()      const {return fActiveMass;}
+
+  // unable SetUnitsOfG, dPhi/dE_signal must have proper units
+  virtual void SetUnitsOfG(Double_t unit);
```

Comment:
On a second thought, the SetUnitsOfG should not be deactivated, because who knows if needed to convert e.g. to <sv>*J or <sv> in different units. I will also change that in the FermiTables class

Reply:
I see you did not do this because "so far we didn't change the data format", which I think is not a good reason. Even for a data format consisting on <sv> values the units are not univoque, so in that sense I think the active version of SetUnitsOfG is needed